library/src/main/java/org/apache/apex/malhar/lib/window/impl/KeyedWindowedOperatorImpl.java

```diff
@@ -187,4 +230,14 @@ public void fireRetractionTrigger(Window window, boolean fireOnlyUpdatedPanes)
     }
   }
 
+  public SpillableSetMultimapImpl<Window, KeyT> getUpdatedDataStorage()
+  {
+    return updatedKeyStorage;
+  }
+
+  public void setUpdatedDataStorage(SpillableSetMultimapImpl<Window, KeyT> updatedDataStorage)
```

Comment:
This is not called anywhere. Can you add a unit test for updatedKeyStorage usage?